hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapAssignAndGetUuidsOperation.java

```diff
@@ -25,14 +25,19 @@
 import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
 import com.hazelcast.spi.impl.operationservice.Operation;
 import com.hazelcast.spi.impl.operationservice.PartitionAwareOperation;
-import com.hazelcast.spi.impl.AllowedDuringPassiveState;
+import com.hazelcast.spi.impl.operationservice.ReadonlyOperation;
 
 import java.util.UUID;
 
 import static com.hazelcast.map.impl.MapService.SERVICE_NAME;
 
+// This operation is used by eventually consistent near cache feature.
+// It creates and gets partition uuid for this purpose. Even we create
+// uuid with this operation, we mark it as a read-only-operation. So,
+// for instance, if number of nodes in a cluster is under configured
+// quorum size, we still can read data for near caching purposes.
 public class MapAssignAndGetUuidsOperation extends Operation
-        implements PartitionAwareOperation, IdentifiedDataSerializable, AllowedDuringPassiveState {
+        implements PartitionAwareOperation, ReadonlyOperation, IdentifiedDataSerializable {
```

Comment:
`Readonly` operations might get rejected if quorum type is `QuorumType#READ` or `QuorumType#READ_WRITE`. For internal operations, it's best to simply remove the `MutatingOperation` or `ReadonlyOperation` or use them and additionally implement `QuorumCheckAwareOperation`. Same for `EntryOffloadableSetUnlockOperation` and

Reply:
done.